Sources/swift-composable-architecture-benchmark/ViewStore.swift

```diff
@@ -1,5 +1,4 @@
 import Benchmark
-import Combine
```

Comment:
Why delete instead of do the `#if canImport` dance?

Reply:
Fair, will fix

Reply:
Oh actually, this is in the benchmark project and this file actually just doesn't need a Combine import at all which is why I removed it.